cmd/import.go

```diff
@@ -111,6 +116,41 @@ func (cmd *ImportDataCmd) handleCsv(ctx context.Context, infoSchema *spanner.Inf
 
 }
 
+func (cmd *ImportDataCmd) handleDatabaseDumpFile(ctx context.Context, dbUri, sourceFormat string, dialect string, spannerAccessor spanneraccessor.SpannerAccessor) error {
+
+	importDump, err := import_file.NewImportFromDump(cmd.project, cmd.instanceId, cmd.databaseName, cmd.sourceUri, sourceFormat, spannerAccessor)
+	if err != nil {
+		return err
+	}
+
+	defer importDump.Finalize()
+
+	schemaStartTime := time.Now()
+	conv, err := importDump.CreateSchema(dialect)
+	if err != nil {
+		return fmt.Errorf(fmt.Sprintf("can't create schema: %v\n", err))
+	}
+
+	// TODO: Only update database
+	err = spannerAccessor.CreateOrUpdateDatabase(ctx, dbUri, sourceFormat, conv, sourceFormat)
+	spannerAccessor.Refresh(ctx, dbUri)
```

Comment:
Is there a use-case for this? `Refresh` was added due to a vey specific limitation with expression verification and I'm not sure if it applies here.

Reply:
If spanner database is created in CreateOrUpdateDatabase, then. the SpannerClient within SpannerAccessor becomes stale. 

Hence, refresh is required.